Wsl.psm1

```diff
@@ -272,6 +272,34 @@ function Get-WslDistribution
     }
 }
 
+<#
+.EXTERNALHELP
+Wsl-help.xml
+#>
+# Retrieves listing of distributions available from online sources
+function Get-WslDistributionOnline {
+    [CmdletBinding()]
+    param(
+    )
+
+    process {
+        $store = $false
+        Invoke-Wsl "--list", "--online" -IgnoreErrors | Select-Object -Skip 1 | ForEach-Object {

```

Comment:
The skip seems unnecessary since you're checking for the header by name.

Reply:
Removed and will test, but suspect you are correct.